docs/api/core/Object3D.html

```diff
@@ -53,21 +53,25 @@
 		<div>
 		Object's local position.
 		</div>
+		<div>
+		readonly – Use methods and properties of the [page:Vector3] class to update the position.
```

Comment:
That is very true.
It's just not directly assignable.

I changed it now to instead explain that.